CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+# Contributing
+
+Thank you for your interest in `tools-python`. The project is open-source software, and bug reports, suggestions, and most especially patches are welcome.
+
+## Issues
+
+`tools-python` has a [project page on GitHub](https://github.com/spdx/tools-python/) where you can [create an issue](https://github.com/spdx/tools-python/issues/new) to report a bug, make a suggestion, or propose a substantial change or improvement. You may also wish to contact the SPDX working group technical team through its mailing list, [spdx-tech@lists.spdx.org](mailto:spdx-tech@lists.spdx.org).
+
+If you would like to work on a fix for any issue, please assign the issue to yourself or write a comment indicating your intention prior to creating a patch.
+
+## Development process
+
+We use the GitHub flow that is described here: https://guides.github.com/introduction/flow/
+
+Here's the process to make changes to the codebase:
+
+0. Find or [file an issue](#issues) you'd like to address. Every change should be made to fix or close an issue.
+
+1. Review [open pull requests](https://github.com/spdx/tools-python/pulls) before committing time to a substantial revision. Work along similar lines may already be in progress.
+
+1. Create a new branch:
```

Comment:
Interesting that the markdown converts this to an ascending numbered lists. For those who read the plain text it might be good to also use ascending numbers here.